api.md

```diff
@@ -2253,23 +2253,25 @@ Get metadata and settings information for continuous aggregates.
 |`refresh_lag` | Amount by which the materialization for the continuous aggregate lags behind the current time |
 |`refresh_interval` | Interval between updates of the continuous aggregate materialization|
 |`max_interval_per_job` | Maximum amount of data processed by a materialization job in a single run|
+|`ignore_invalidation_older_than` | Interval for which updated rows will trigger update of the continuous aggregate|
```

Comment:
typo -> will not
```suggestion
|`ignore_invalidation_older_than` | Interval for which modified rows (i.e. insert/update/delete) will not trigger update of the continuous aggregate|
```

Reply:
Shouldn't it be "will" since updates in the interval [NOW - IIOT, NOW] will trigger an update, or have I misunderstood how it works?

Reply:
I think of it as ignoring anything before NOW - IIOT. Maybe we should not refer to it as _interval_ as it is not clear which interval we are referring to : is it anything < NOW - IIOT or the other one.

Reply:
Yes, but not sure how to re-write it without using active tense. "Age before which modifications do not trigger an update of the continuous aggregate."?

Reply:
lets do that and see if anyone else has suggestions.